pkg/controllers/cloud/networkpolicy_controller.go

```diff
@@ -487,6 +515,10 @@ func (r *NetworkPolicyReconciler) Start(stop context.Context) error {
 		r.Log.Error(err, "Start watchers")
 	}
 
+	// TODO: remove sleep and implement re-sync on controller restart
```

Comment:
move this to line 486 and mention, introducing a sleep of 10 sec to allow CPA and CES controller to reconcile CRs